field/composite_test.go

```diff
@@ -95,6 +95,96 @@ var (
 		},
 	}
 
+	compositeTestSpecWithDefaultBitmap = &Spec{
+		Length:      32,
+		Description: "Test Spec",
+		Pref:        prefix.ASCII.LL,
+		Bitmap: NewBitmap(&Spec{
+			Length:      8,
+			Description: "Bitmap",
+			Enc:         encoding.BytesToASCIIHex,
+			Pref:        prefix.Hex.Fixed,
+		}),
+		Subfields: map[string]Field{
+			"2": NewString(&Spec{
+				Length:      2,
+				Description: "String Field",
+				Enc:         encoding.ASCII,
+				Pref:        prefix.ASCII.LL,
+			}),
+			"3": NewNumeric(&Spec{
+				Length:      2,
+				Description: "Numeric Field",
+				Enc:         encoding.ASCII,
+				Pref:        prefix.ASCII.LL,
+			}),
+			"11": NewComposite(&Spec{
+				Length:      6,
+				Description: "Sub-Composite Field",
+				Pref:        prefix.ASCII.LL,
+				Tag: &TagSpec{
+					Length: 2,
+					Enc:    encoding.ASCII,
+					Pad:    padding.Left('0'),
+					Sort:   sort.StringsByInt,
+				},
+				Subfields: map[string]Field{
+					"1": NewString(&Spec{
+						Length:      2,
+						Description: "String Field",
+						Enc:         encoding.ASCII,
+						Pref:        prefix.ASCII.LL,
+					}),
+				},
+			}),
+		},
+	}
+
+	compositeTestSpecWithSizedBitmap = &Spec{
+		Length:      30,
+		Description: "Test Spec",
+		Pref:        prefix.ASCII.LL,
+		Bitmap: NewBitmap(&Spec{
+			Length:      3,
```

Comment:
Let's add `DisableAutoExpand: true` to avoid bitmap expanding

Reply:
Done